### ISSUE TITLE
Use organization short name in dropdowns

### ISSUE THREAD

**Issue by eikek** [eikek/docspell#830]
See https://app.element.io/#/room/#eikek_docspell:gitter.im/$EJE4ad7qpnnMFWr5MfysJYrEXGx2VWYNfq_H7pG1H2I

The organization dropdown and maybe suggestion list can use the organization's short name if that is available

**Comment by eikek** [comment]
There are multiple places where the organization name is shown:

- dropdown
- suggestions
- item card
- item detail

I'd use the short name only in dropdowns - and maybe suggestions - but the other places I think the longer name is better?

**Comment by monnypython** [comment]
I agree with you. The short name should only be used for searching. The result should always be the full name.
